chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -900,7 +901,7 @@ public synchronized void loadChunks(TaskTracker taskTracker, World world, Collec
       for (int i = 0; i < chunkPositions.length; i++) {
         ChunkPosition cp = chunkPositions[i];
 
-        task.update(target, done);
+        task.update(target, done, startTime);
```

Comment:
```suggestion
        task.update(target, done);
```

Putting the start time in every update is strange… Instead you could set it automatically when the task starts, i.e. when `taskTracker.task()` is called and the `Task` is created.